packages/access/lib/grants.js

```diff
@@ -47,6 +50,24 @@ const evaluateConstraints = async (granter, campaign, email, t, pgdb) => {
   return { errors }
 }
 
+const generate = async (granter, campaignId, grants = [], pgdb) => {
```

Comment:
This function doesn't generate anything, IMHO `insert` would be a more appropriate name.

Reply:
Yes, `insert` is better.